ims-react/src/components/TableSortGroup.js

```diff
@@ -1,11 +1,21 @@
 function descendingComparator(a, b, orderBy) {
-  if (b[orderBy] < a[orderBy]) {
-    return -1;
+  if(orderBy.toLowerCase().includes('date')){
+    if (new Date(b[orderBy]).getTime() < new Date(a[orderBy]).getTime()) {
```

Comment:
I think these conditional validations could more self-documenting by breaking them each out into functions that are named what they do?  I would also give the variables `a` and `b` real names to help a future developer understand what is going on quicker.